### PR TITLE
update pointer's docs

### PR DESCRIPTION
**Pull request by AmosChenYQ**

### CODE REVIEW

**Review by susiwen8 — CHANGES_REQUESTED** [review]
please add version, this feature will be included in 5.2.0

File: en/option/series/gauge.md [hunk 1]
```diff
@@ -295,6 +295,12 @@ Gauge chart pointer.
 
 Whether to show the pointer.
 
+### showAbove(boolean) = true
```
Comment (susiwen8) [inline en/option/series/gauge.md:298]: You have misunderstood. It's not changelog you need to add. You should add `{{ use: partial-version( version = "5.2.0" ) }}` at here, there are serval examples in `gauge.md`

**Review by susiwen8 — CHANGES_REQUESTED** [review]
You have misunderstood. It's not changelog you need to add. You should add `{{ use: partial-version( version = "5.2.0" ) }}`, there are serval examples in `gauge.md`

**Review by susiwen8 — APPROVED** [review]

**Comment by pissang** [comment]
LGTM

**Pull request merged**